style/servo/selector_parser.rs

```diff
@@ -173,10 +181,11 @@ impl PseudoElement {
         false
     }
 
-    /// Whether this pseudo-element is the ::-moz-color-swatch pseudo.
+    /// Whether this pseudo-element is the pseudo element representing
+    /// the color swatch inside an `<input>` element.
```

Comment:
Seems a bit weird to repeat "pseudo-element", especially since one has an hyphen and the order doesn't.

```suggestion
    /// Whether this pseudo-element is representing the color swatch
    /// inside an `<input>` element.
```